docs/ml-features.md

```diff
@@ -1064,7 +1069,8 @@ categorical features.
 The bin ranges are chosen by taking a sample of the data and dividing it into roughly equal parts.
 The lower and upper bin bounds will be `-Infinity` and `+Infinity`, covering all real values.
 This attempts to find `numBuckets` partitions based on a sample of the given input data, but it may
-find fewer depending on the data sample values.
+find fewer depending on the data sample values. Relative precision of the approxQuantile is set using
```

Comment:
@oliverpierson actually in looking at adding the PySpark I also have just come across this doc issue on both the Scala and Python sides. I will submit a PR soon and ping you on it.